configs/mask_rcnn/neptune_config.py

```diff
@@ -0,0 +1,22 @@
+_base_ = [
```

Comment:
Is this file needed?

Reply:
Yes, it is what actually runs the hook I guess. Other integrations also have it  (e.g. https://github.com/AleksanderWWW/mmdetection/blob/master/configs/mask_rcnn/mask_rcnn_r50_fpn_1x_wandb_coco.py ) - though we could probably come up with a better name

Reply:
I see.